deploy/lib/server_config.rb

```diff
@@ -1046,6 +1046,8 @@ def deploy
         deploy_triggers
       when 'rest-config'
         deploy_rest_config
+      when 'config'
+        deploy_config
```

Comment:
By adding these lines, you're making "ml {env} deploy config" a command line option. I see this more as a modification to the existing deploy_src functionality than a new deploy option. (If we were going to make it an option, it would need to be added to the Help as well.) 

I think this should be dropped.